pkg/operator/context.go

```diff
@@ -47,7 +46,7 @@ type Context struct {
 
 	EventRecorder     events.Recorder      // Decorated recorder for Karpenter core events
 	BaseEventRecorder record.EventRecorder // Recorder from controller manager for use by other components
-	Config            config.Config
+	ConfigMapWatcher  *informer.InformedWatcher
```

Comment:
Not seeing where this is used

Reply:
This is going to be used when we hoist up the initialization of the settings in the `cmd/main` of the controller in the `Karpenter` project. This needs to be there before doing that.